src/javascript/app/base/footer.js

```diff
@@ -27,6 +28,42 @@ const Footer = (() => {
         const $status_notification = $('#status_notification');
         $status_notification.slideUp(200);
     };
+
+    const adjustElevioAndScrollup = (elevio_height, scrollup_height) => {
+        const $elevio_button = $('#_elev_io ._6byvm');
+        const $scrollup = $('#scrollup');
+        $elevio_button.attr('style', `bottom: ${elevio_height}px !important`);
+        $scrollup.attr('style', `bottom: ${$elevio_button.height() + scrollup_height}px`);
+    };
+
+    const clearDialogMessage = () => {
+        const $dialog_notification = $('#dialog_notification');
+
+        $dialog_notification.slideUp(200);
+        adjustElevioAndScrollup(8, 18);
+    };
+
+    const displayDialogMessage = () => {
+        BinarySocket.wait('website_status', 'authorize', 'landing_company').then(() => {
+            if (isEuCountry()) {
+                const $dialog_notification = $('#dialog_notification');
+                const $dialog_notification_accept = $('#dialog_notification_accept');
```

Comment:
can we use javascript for this since it doesn't require any jquery animations?

Reply:
you mean using document.geteElementById('###') ?